pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -446,6 +446,73 @@ func TestPriorityQueue_Activate(t *testing.T) {
 	}
 }
 
+const queueable = "queueable"
+
+type preEnqueuePlugin struct {
+	allowlists []string
+}
+
+func (pl *preEnqueuePlugin) Name() string {
+	return "preEnqueuePlugin"
+}
+
+func (pl *preEnqueuePlugin) PreEnqueue(ctx context.Context, p *v1.Pod) *framework.Status {
+	for _, allowed := range append(pl.allowlists, queueable) {
```

Comment:
I am not sure if i understand correctly：maybe we can delete the default allow name `queueable` to make the test understood easily.
And change the case from
```go
  plugins:  []framework.EnqueuePlugin{&preEnqueuePlugin{allowlists: []string{"foo"}}, &preEnqueuePlugin{}},
- pod:      st.MakePod().Name(queueable).Obj(),
+ pod:      st.MakePod().Name("foo").Obj(),
```

Reply:
udpated.